react-loops.test.js

```diff
@@ -89,7 +89,15 @@ describe("react-loops", () => {
         </>
       );
     });
-
+    it("loops fallback", () => {
```

Comment:
Thanks for adding a test! Can you also test for `{}` in a `for-in` and `null` in both cases?

Also, what about a test if `fallback` is not a function?

Reply:
I will fix it.